test/extensions/filters/http/ext_authz/ext_authz_fuzz.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package envoy.extensions.filters.http.ext_authz;
+
+import "envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto";
+import "test/fuzz/common.proto";
+import "envoy/config/core/v3/base.proto";
+import "google/protobuf/empty.proto";
+import "validate/validate.proto";
+
+message Result {
+  oneof result_selector {
+    option (validate.required) = true;
+    // Authorization check status
```

Comment:
Maybe provide an explicit reference to the enum class this is based on (`source/extensions/filters/common/ext_authz/ext_authz.h`) so it's clear what these are for.

Reply:
Could even call them `OK`, `Error`, `Denied` to match the underlying `enum class`.